daml-script/export/BUILD.bazel

```diff
@@ -111,3 +119,65 @@ da_scala_test(
         "@maven//:io_netty_netty_handler",
     ],
 )
+
+da_scala_binary(
+    name = "example-export-client",
+    srcs = ["src/example-export/scala/com/daml/script/export/ExampleClient.scala"],
+    main_class = "com.daml.script.export.ExampleClient",
+    scala_deps = [
+        "@maven//:com_github_scopt_scopt",
+    ],
+    deps = [
+        ":export",
+        "//daml-lf/data",
+        "//daml-script/runner:script-runner-lib",
+        "//language-support/scala/bindings",
+        "//language-support/scala/bindings-akka",
+        "//ledger/ledger-api-common",
+        "//libs-scala/auth-utils",
+    ],
+)
+
+client_server_build(
+    name = "example-export",
+    outs = [
+        "example-export/Export.daml",
+        "example-export/args.json",
+    ],
+    client = ":example-export-client",
+    client_files = ["//daml-script/test:script-test.dar"],
+    data = ["//daml-script/test:script-test.dar"],
+    output_env = "EXPORT_OUT",
+    server = "//ledger/sandbox:sandbox-binary",
+    server_files = ["//daml-script/test:script-test.dar"],
+)
+
+# Compare the generated Daml ledger export to the example export used in the
+# documentation. This functions as both a golden test on ledger exports and to
+# make sure that the documentation stays up-to-date.
+sh_inline_test(
+    name = "example-export-compare",
+    cmd = """\
+EXPECTED_EXPORT=$$(canonicalize_rlocation $(rootpath //docs:source/tools/export/output-root/Export.daml))
+EXPECTED_ARGS=$$(canonicalize_rlocation $(rootpath //docs:source/tools/export/output-root/args.json))
+ACTUAL_EXPORT=$$(canonicalize_rlocation $(rootpath :example-export/Export.daml))
+ACTUAL_ARGS=$$(canonicalize_rlocation $(rootpath :example-export/args.json))
+# Normalize the expected file by removing the copyright header and any documentation import markers.
+# Normalize the actual output by adding a newline to the last line if missing.
```

Comment:
Should these comments go on top of the test rule? Spotting comments on inlined scripts can be tricky. Duplicating the comment is probably fine.

Reply:
That's a good point. I've expanded the top comment and kept the comment in the inline code.